appveyor.yml

```diff
@@ -72,6 +88,9 @@ install:
     # Remove cygwin (and therefore the git that comes with it).
     - cmd: rmdir C:\cygwin /s /q
 
+    # Install workaround 64-bit support for VS 2008.
+    - cmd: if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "27" call .CI\vs2008\setup_x64.bat
```

Comment:
is this now fixed? I just copied what other PR seems to do, because the x64 build didn't even start.